docs/openapi/schemas.yaml

```diff
@@ -177,17 +176,16 @@ OrganizationCreate:
       slack:
         channel: '#example-channel'
         workspace: 'example-workspace'
-      alerts:
-        - type: '404'
-          byOrg: true
+      handlers:
```

Comment:
We can start reusing examples, when they become repetitive: https://swagger.io/docs/specification/adding-examples/ -> Reusing Examples